assets/js/app/Instance.js

```diff
@@ -43,6 +46,14 @@ LocusZoom.Instance = function(id, datasource, layout, state) {
 
 LocusZoom.Instance.prototype.initializeLayout = function(){
 
+    // If this is a responsive layout then set a namespaced/unique onresize event listener on the window
+    if (this.layout.resizable == "responsive"){
+        this.window_onresize = d3.select(window).on("resize.lz-"+this.id, function(){
```

Comment:
Note that this is a "namespaced" event listener. I didn't even know you could do this until today, but essentially the ".lz-" and the id following the "resize" still ties this method to a resize event on the window but does *not* affect any other existing resize event listeners.